src/modules/game/BatteryPercent.js

```diff
@@ -3,19 +3,19 @@ class BatteryPercent extends HFX.Feature {
     super({
       section: HFX.Section.Game,
       name: "Battery Percent",
-      default: 1,
+      default: true,
       description: "Show percentage next to battery icon.",
       id: "batterypercent"
     });
   }
 
   run() {
     if ($("#game_content_currentpage").length > 0) {
-      var batteryPercent = (
+      const batteryPercent = (
```

Comment:
Should I use const anywhere possible compared to var or let?

Reply:
It's not too important in short code, but if the variable is constant it's good to use `const` and otherwise `let`. Doing so helps make sure variable scoping is correct and that variables aren't being reinitialized/altered on accident.